crates/common/download/Cargo.toml

```diff
@@ -28,4 +28,5 @@ mockito = "0.31"
 regex = "1.5"
 tempfile = "3.2"
 test-case = "2.2"
+tokio = { version = "1.23", features = ["macros"] }
```

Comment:
This seems to be added by mistake.

Reply:
I added this as without it, you can't run tests for just this crate, and I was trying to debug why the tests were failing for me locally.